src/styles/shared.scss

```diff
@@ -8,6 +8,9 @@ body {
 	background-color: colorpurple;
 	color: $white;
 }
+.pg {
```

Comment:
I strongly believe you can find a better name for this selector.

Reply:
Alright. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198153%2Fshare-article/src%2Fstyles%2Fshared.scss/#L11-L13